src/OpenTelemetry.Contrib.Instrumentation.AWS/OpenTelemetry.Contrib.Instrumentation.AWS.csproj

```diff
@@ -1,18 +1,21 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net452;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net462;netstandard2.0</TargetFrameworks>
     <Description>AWS client instrumentation for OpenTelemetry .NET</Description>
 	<MinVerTagPrefix>Instrumentation.AWS-</MinVerTagPrefix>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="OpenTelemetry.Contrib.Extensions.AWSXRay" Version="1.0.1" />
-    <PackageReference Include="AWSSDK.Core" Version="3.5.1.24" />
+    <PackageReference Include="AWSSDK.Core" Version="3.7.12.20" />
   </ItemGroup>
 
   <ItemGroup>
     <Compile Include="$(RepoRoot)\src\OpenTelemetry.Internal\Guard.cs" Link="Includes\Guard.cs" />
-    <Compile Include="$(RepoRoot)\src\OpenTelemetry.Contrib.Shared\Api\SemanticConventions.cs" Link="Includes\SemanticConventions.cs"/>
+    <Compile Include="$(RepoRoot)\src\OpenTelemetry.Contrib.Shared\Api\SemanticConventions.cs" Link="Includes\SemanticConventions.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\OpenTelemetry.Contrib.Extensions.AWSXRay\OpenTelemetry.Contrib.Extensions.AWSXRay.csproj" />
```

Comment:
I think that it is not safe change. Each of AWS* package is released independently based on the tags `Extensions.AWSXRay-`, `Instrumentation.AWSLambda-`, `Instrumentation.AWS-`.

I suppose that releasing process is pretty manual and requires at release AWSXRay and then PR + additional release for other libs.

Reply:
Right. With this change, things will break unless all of the cross-referenced packages are always released at the same time, which is not something that is enforced now.

Also the Lambda package has a different component owner from the other packages, and it's likely we'll request a release soon, without wanting to (or even having the responsibility to) release the other packages.

Reply:
Alright so I reverted that part of the PR.  It builds and test pass, but I was a little concerned because the XRay package referenced causes a transitive dependency to a lower version of OpenTelemetry.  As such, it would probably be worthwhile to the referenced version of XRay just as soon as it is released.  In the mean time though it doesn't seem to break anything at least.

Reply:
In these cases, the higher version should be used, and if that breaks anything, it would be a bug, because 1.x versions are guaranteed to be backwards compatible.